src/RealtimeClient.ts

```diff
@@ -360,6 +357,14 @@ export default class RealtimeClient {
     }
   }
 
+  private _resetHeartbeat() {
+    this.pendingHeartbeatRef = null
+    clearInterval(this.heartbeatTimer)
+    this.heartbeatTimer = <any>(
```

Comment:
For compatibility with the node.js is better to use <ReturnType<typeof setInterval>>

Reply:
Agreed - seems to be the recommended approach online

Reply:
Let me know what you think of https://github.com/supabase/realtime-js/pull/80/commits/3508f49556056f06a9336730be5a46d12ed91506.

The alternative was to do `heartbeatTimer!: ReturnType<typeof setInterval>` but I thought this way was better.

Reply:
this looks good!

Reply:
Good for me too

Reply:
Thanks @kiwicopple and @abc3!